src/components/cylc/tree/event-bus.js

```diff
@@ -0,0 +1,3 @@
+import Vue from 'vue'
+
+export const TreeEventBus = new Vue()
```

Comment:
It is this simple? Wow!

Reply:
Yup. From what I remember (no access to code right now), the components we create become `VueComponent`, which extend `Vue`.

In the same way that within a `VueComponent` we are able to simply do a `this.$emit`, and call `@on:click=this.method`, we can instead create a new instance of `Vue` (parent of `VueComponent`) and use just the event handling part.